iron/distribution.yaml

```diff
@@ -1731,6 +1731,24 @@ repositories:
       url: https://github.com/borglab/gtsam.git
       version: develop
     status: developed
+  gz_ros2_control:
+    doc:
+      type: git
+      url: https://github.com/ros-controls/gz_ros2_control.git
+      version: master
+    release:
+      packages:
+      - gz_ros2_control
+      - gz_ros2_control_demos
+      tags:
+        release: release/iron/{package}/{version}
+      url: https://github.com/ros2-gbp/ign_ros2_control-release.git
+      version: 1.1.0-1
```

Comment:
Something is weird here; this version does not exist in https://github.com/ros2-gbp/ign_ros2_control-release , and that is why CI is failing.